tests/cupyx_tests/scipy_tests/ndimage_tests/test_interpolation.py

```diff
@@ -327,6 +328,7 @@ class TestAffineTransformTextureMemory:
     _multiprocess_can_split = True
 
     def _2d_rotation_matrix(self, theta, rotation_center):
+        import scipy.special
```

Comment:
I wonder why the CI didn't catch this. I am on SciPy 1.5.2 and without this line the actual error was covered up by `AttributeError: module 'scipy' has no attribute 'special'` instead.

Reply:
Good catch! I think you will only see that error when running this test alone. It won't happen if other tests in the same process already imports `scipy.special`.